### ISSUE TITLE
Image doesn't support id attribute

### ISSUE THREAD

**Issue by MrCreeper1008** [dnfield/flutter_svg#274]
I have a `use` tag in my svg:  
```svg
<use xlink:href="#_Image2" x="0.073" y="0" width="47.999px" height="64px" transform="matrix(0.99998,0,0,1,0,0)"/>
```  
which is referencing:
```svg
<image id="_Image2" width="48px" height="64px" xlink:href="data:image/png;base64,..."/>
```  
defined inside topmost `<defs>`, but it causes:  
```
Bad state: Expected to find Drawable with id url(#_Image2).
```  

Stack:
```
When the exception was thrown, this was the stack
#0      DrawableDefinitionServer.getDrawable 
package:flutter_svg/src/vector_drawable.dart:581
#1      _Elements.use 
package:flutter_svg/…/svg/parser_state.dart:148
#2      SvgParserState.parse 
package:flutter_svg/…/svg/parser_state.dart:713
<asynchronous suspension>
#3      SvgParser.parse 
package:flutter_svg/parser.dart:14
...
Picture provider: ExactAssetPicture(name: "assets/svg/colored-icon-light-theme.svg", bundle: null, colorFilter: null)
Picture key: AssetBundlePictureKey(bundle: PlatformAssetBundle#59019(), name: "assets/svg/colored-icon-light-theme.svg", colorFilter: null)
```

**Comment by MrCreeper1008** [comment]
The svg is generated by Affinity Designer.

**Comment by dnfield** [comment]
I think this is the same as #273

**Comment by MrCreeper1008** [comment]
The original issue is having issue with a use tag referencing another use tag, which I think is not same as this issue. Am I missing anything?

**Comment by dnfield** [comment]
Sorry I mixed up the order.  It would help if you shared a full reproduction

**Comment by MrCreeper1008** [comment]
Will do later today. Thanks!

**Comment by MrCreeper1008** [comment]
Due to [this issue](https://github.com/flutter/flutter/issues/46792#issuecomment-576066956), I'm unable to create a reproduction now until it's resolved. I do have a small svg file that leads to this issue though.

**Comment by dnfield** [comment]
Just the svg is enough

**Comment by MrCreeper1008** [comment]
```svg
<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
<svg width="100%" height="100%" viewBox="0 0 64 64" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" xml:space="preserve" xmlns:serif="http://www.serif.com/" style="fill-rule:evenodd;clip-rule:evenodd;stroke-linejoin:round;stroke-miterlimit:2;">
    <defs>
        <image id="_Image1" width="38px" height="34px" xlink:href="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACYAAAAiCAYAAAAzrKu4AAAACXBIWXMAAA7EAAAOxAGVKw4bAAAA/UlEQVRYhe2YsW7DMAxEnxUVddohBbr0/78waZHEgWO5g06xVi5MBh5wkE0PfKC88AZgABKw05lU89QKFHkBShbIG7AHRj17wjWoGZiAKzBn6qT2wJf8IThPsBm4AEfVHhMbBfUDHIB3Z7AbcNL7BFz7q/wU1LdAkxNYEQzAn1hSZvv5M3VSI/VqPSeGerdBDVnFoXPqTg+VrmezW3OzAsyqALMqwKwKMKsCzKoAsyrArAowqwLMqgCzqm1Ja+fSnR7qezaTuw936kbclk/vhfcmhgKsmS3QOLOt6c+KCM5iKQ1sYgs0fnluqDI1sIUa/aDiK8RQSx8LvFRw9w+YoVIODOGLegAAAABJRU5ErkJggg=="/>
    </defs>
    <use xlink:href="#_Image1" x="13.07" y="15.079" width="37.917px" height="33.917px" transform="matrix(0.997807,0,0,0.997549,0,0)"/>
    <g transform="matrix(1,0,0,1,-5.5,5.5)">
        <rect x="21" y="12" width="33" height="29" style="fill:rgb(235,235,235);"/>
    </g>
</svg>
```

**Comment by dnfield** [comment]
Fixed on master now - will be published once CI finishes as v0.16.1

**Comment by dnfield** [comment]
Published as v0.16.1

**Comment by MrCreeper1008** [comment]
Thank you, will test this today.
